.gitignore

```diff
@@ -22,4 +22,5 @@
 hs_err_pid*
 
 .idea/
-*.iml
\ No newline at end of file
+*.iml
+target/
```

Comment:
I added `target/` to the `.gitignore` file. This file contains `*.class` files and other compiled output that other people can just use `mavenw` to create on their own machines, so there's no point in checking them into version control.